network.go

```diff
@@ -890,3 +893,17 @@ func RunNet(i interface{}) {
 		close(net.done)
 	}()
 }
+
```

Comment:
Websocket code is a part of the protocol runtime, it should not be present in `network.go` which can also be used in runtime-less applications.